Makefile

```diff
@@ -312,3 +312,13 @@ push-plugin: build-plugin
 
 enable-plugin:
 	docker plugin enable grafana/loki-docker-driver:$(PLUGIN_TAG)
+
+BIGTABLE_BACKUP_TOOL_FOLDER = ./tools/bigtable-backup
+BIGTABLE_BACKUP_TOOL_TAG ?= $(IMAGE_TAG)
+
+build-bigtable-backup-tool:
+	docker build -t $(IMAGE_PREFIX)$(shell basename $(BIGTABLE_BACKUP_TOOL_FOLDER)) $(BIGTABLE_BACKUP_TOOL_FOLDER)
+	docker tag $(IMAGE_PREFIX)$(shell basename $(BIGTABLE_BACKUP_TOOL_FOLDER)) $(IMAGE_PREFIX)loki-bigtable-backup-tool:$(IMAGE_TAG)
+
+push-bigtable-backup-tool: build-bigtable-backup-tool
+	docker plugin push $(IMAGE_PREFIX)loki-bigtable-backup-tool:$(IMAGE_TAG)
```

Comment:
is it a docker plugin ?

Reply:
No, sorry my bad. Haven't tried pushing it since the repository is not setup yet. Will fix it.
Thanks!